README.md

```diff
@@ -274,6 +274,19 @@ EOF
 You can also skip step 2 and use `:TSInstallFromGrammar zimbu` to install directly from a `grammar.js` in the top-level directory specified by `url`.
 Once the parser is installed, you can update it (from the latest revision of the `main` branch if `url` is a Github repository) with `:TSUpdate zimbu`.
 
+## Update parsers used_by
+
+Sometimes needs to use some parser for different filetype.
+
+Add the following snippet to your `init.vim`:
```

Comment:
This is actually super helpful for me, I appreciate it!
The text on this section seems to be lacking some context and why people would want to do this.

After reading this: https://github.com/nvim-treesitter/nvim-treesitter/issues/31

Wouldn't people need to have this code on their vim configs in order to make this change work?
```
au! BufRead,BufNewFile *.js if getline(1) =~ '//\s*@flow' | setf javascriptflow | endif
```